pkg/querier/frontend/worker.go

```diff
@@ -31,9 +31,9 @@ var (
 
 // WorkerConfig is config for a worker.
 type WorkerConfig struct {
-	Address           string
-	Parallelism       int
-	DNSLookupDuration time.Duration
+	Address           string        `yaml:"address"`
```

Comment:
What is address? URL? Can be `frontend_url` instead

Reply:
We should specify expected format of address / host fields, especially if it is not just hostname.